Namespace(batchSize=16, cuda=True, dataset='datasets/', lr=0.0001, momentum=0.9, nEpochs=5000, resume='', start_epoch=1, step=1000, test_path='dataset/test/', threads=16, validate_path='datasets/validate/', weight_decay=0)
===> Loading datasets
===> Building model
n_pyramids: 1 n_pyramid_cells: (3, 2, 1, 1, 1, 1)
===> Setting GPU
===> Setting Optimizer
===> Training
Epoch=1, lr=0.0001
===> Epoch[1](1/11): Loss: 0.5134869814
===> Epoch[1](2/11): Loss: 0.3545281291
===> Epoch[1](3/11): Loss: 0.3398526907
===> Epoch[1](4/11): Loss: 0.3559057713
===> Epoch[1](5/11): Loss: 0.3986468911
===> Epoch[1](6/11): Loss: 0.4111340940
Traceback (most recent call last):
  File "train_psnr_indication.py", line 331, in <module>
    main()
  File "train_psnr_indication.py", line 160, in main
    validate(model, optimizer, epoch)
  File "train_psnr_indication.py", line 290, in validate
    validate_data_loader = DataLoader(dataset=validate_set, batch_size=1, shuffle=True)
  File "/home/pxk/anaconda3/envs/cxy/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 266, in __init__
    sampler = RandomSampler(dataset, generator=generator)  # type: ignore
  File "/home/pxk/anaconda3/envs/cxy/lib/python3.7/site-packages/torch/utils/data/sampler.py", line 102, in __init__
    if not isinstance(self.num_samples, int) or self.num_samples <= 0:
  File "/home/pxk/anaconda3/envs/cxy/lib/python3.7/site-packages/torch/utils/data/sampler.py", line 110, in num_samples
    return len(self.data_source)
  File "/home/pxk/CBREN/dataset.py", line 156, in __len__
    return len(self.lis)
AttributeError: 'ValidateDataset' object has no attribute 'lis'
===> Epoch[1](7/11): Loss: 0.4329962730
===> Epoch[1](8/11): Loss: 0.3918805718
===> Epoch[1](9/11): Loss: 0.3571768999
===> Epoch[1](10/11): Loss: 0.2859341502
===> Epoch[1](11/11): Loss: 0.3429052234
|||||||||||||||||||||min_epoch_loss is 0.3804043342|||||||||||||||||||||
----Start Validate----